### ISSUE TITLE
vim8 and neovim now has QuickFixLine

### ISSUE THREAD

**Issue by peterhellberg** [nanotech/jellybeans.vim#67]
The quickfix line used to use he `Search` highlight settings, now it has a separate setting called `QuickFixLine`.

I believe the easiest way to have the old look when using the jellybeans theme with current versions of vim8/neovim is to add the following line after [line 453](https://github.com/nanotech/jellybeans.vim/blob/2ca7d8a27072a8f449887179c3fee81cb00f8c18/colors/jellybeans.vim#L453):

```vim
call s:X("QuickFixLine","f0a0c0","302028","underline","Magenta","")
```

**Comment by peterhellberg** [comment]
Example of ugly quickfix line, as shown when using `Ack.vim`:

![cyan-quickfix](https://assets.c7.se/screenshots/vim-quickfix-list-horrible-color-combination-20170615-130023.png)

Currently fixed by `hi QuickFixLine guibg=#302028 guifg=#f0a0c0 cterm=underline` in my **.vimrc**

**Comment by nanotech** [comment]
This was [fixed in Vim 8.0.0653](https://github.com/vim/vim/commit/c768a208ca8e0e0fec900c18d5d9a593357ad793) by linking `QuickFixLine` to `Search`. Thanks for the report!
